docs/tech_primer.md

```diff
@@ -20,20 +20,27 @@ For the most part, this document addresses the scope of a Smart Ready building:
 
 ## Device
 
-Hardware that exists in the building that can change state in a way that can be machine communicated. A potentiometer or a flow switch is a device if it is monitored by a digital system. A commissioning valve or damper with no communication is not a device. 
+Hardware that exists in the building that can change state in a way that can be
```

Comment:
This isn't quite accurate.  A "device" is a logical entity that encapsulates a coordinated set of functionality.  The problem is it's a very over-loaded term and so you can't say much in the way of definitions.  Not all devices are hardware. Not all devices an be machine communicated (although it would be kinda pointless otherwise). 

I'm not sure trying to define "device" will actually get you very far... which is why I jumped strait to the descriptive adjectives.